diff --git a/node_modules/react-native/scripts/packager-reporter.js b/node_modules/react-native/scripts/packager-reporter.js
index dc4066c..2009bc0 100644
--- a/node_modules/react-native/scripts/packager-reporter.js
+++ b/node_modules/react-native/scripts/packager-reporter.js
@@ -27,7 +27,9 @@ if (logPath != null && logPath !== '') {
     }
   };
 } else {
-  reporter = require('metro').TerminalReporter;
+  const metro = require('metro');
+  reporter = metro.TerminalReporter || metro.reporter?.TerminalReporter;
 }
 
+
 module.exports = reporter;
